communication/marketing-team/storytelling-resources/blog-guidelines.md

```diff
@@ -14,9 +14,9 @@ Other types of content, like Kubernetes capabilities and tutorials, are better s
 
 ## Submit a Post
 
-The quickest way to get involved is to let the team in [#sig-contribex](https://kubernetes.slack.com/archives/C1TU9EB9S) know that you have an idea for an article.
+The quickest way to get involved is to let the team in [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20) know that you have an idea for an article.
 
-For now, our official process is to use [SIG-Doc's system](/sig-docs/blog-subproject/README.md).
+For now, our official process is to use [SIG-Doc's system](/sig-docs/blog-subproject/README.md), with one change: instead of directly creating the file in the Kubernetes site repository, as instructed abovem, it's initially created in the [contributor-site](https://github.com/kubernetes/contributor-site), in the appropriate folder (i.e. the right year in `contributor-site/content/en/blog/`). This will lead to an initial review process before it gets mirrored to the main Kubernetes site. 
```

Comment:
```suggestion
For now, our official process is to use [SIG Docs' system](/sig-docs/blog-subproject/README.md), with one change: instead of directly creating the file in the Kubernetes site repository, as instructed abovem it's initially created in the [contributor-site](https://github.com/kubernetes/contributor-site), in the appropriate folder (i.e. the right year in `contributor-site/content/en/blog/`). This will lead to an initial review process before it gets mirrored to the main Kubernetes site.
```

Reply:
Do we want to suggest that people start off in a Google Doc / hackmd, and then open a PR once an initial pass through?